tests/unit/Certify.spec.ts

```diff
@@ -198,23 +198,36 @@ describe('Certify', () => {
     inputElement.setValue(trimmedCertifier)
 
     // The last "update:certifiedBy" event should match the input.
-    expect(getLastEvent(wrapper, 'certifiedByChange')).toMatch(trimmedCertifier)
+    expect(getLastEvent(wrapper, 'certifiedBy')).toMatch(trimmedCertifier)
   })
 
   it('has update event for trimmed certifiedBy', () => {
     const wrapper: Wrapper<Certify> = createComponent(' ')
     const inputElement: Wrapper<Vue> = wrapper.find(certifiedBySelector)
     inputElement.setValue(whitespaceCertifier)
     // The last "update:certifiedBy" event should be a trimmed version of the input.
-    expect(getLastEvent(wrapper, 'certifiedByChange')).toMatch(trimmedCertifier)
+    expect(getLastEvent(wrapper, 'certifiedBy')).toMatch(trimmedCertifier)
   })
 
-  it('has update event for isCertified', () => {
+  it('has update event for isCertified = true', () => {
     const wrapper: Wrapper<Certify> = createComponent()
     const checkboxElement: Wrapper<Vue> = wrapper.find(isCertifiedSelector)
-    checkboxElement.setChecked()
+    // toggle checkbox to guarantee a change event
+    checkboxElement.setChecked(false)
+    checkboxElement.setChecked(true)
```

Comment:
This is because a change event isn't emitted if the property is set to its existing value.